lib/board.rb

```diff
@@ -0,0 +1,52 @@
+require 'pry'
+class Board
+  attr_accessor :cells, :token
+
+
+
+  def reset!
+   @cells = Array.new(9, " ")
+  end
+
+  def initialize
+    reset!
+  end
+
+ def display
+  puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+  puts "-----------"
+  puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+  puts "-----------"
+  puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+ end
+
+def position(input)
+cells[input.to_i - 1]
```

Comment:
Being consistent with the whitespace in your code, especially indentation, will make your code more readable.